application/language/russian/project_launch_lang.php

```diff
@@ -0,0 +1,127 @@
+<?php
+$lang['proj_launch_intro_header'] = 'ЛИБРИВОКС: ГЕНЕРАТОР ШАБЛОНОВ НОВЫХ ПРОЕКТОВ';
+$lang['proj_launch_intro_1'] = 'ВАС ПРИВЕТСТВУЕТ Генератор шаблонов новых проектов Либривокс.';
+$lang['proj_launch_intro_2'] = 'Введите ниже как можно больше информации';
+$lang['proj_launch_intro_3'] = 'Не тревожьтесь, ваш MC сможет внести исправления позже.';
+$lang['proj_launch_intro_4'] = 'Когда закончите, нажмите "Submit" чтобы сгенерировать код, который вы поместите на форум.';
+$lang['proj_launch_intro_5'] = '(*) поля помеченные звездой необходимо заполнить';
+$lang['proj_launch_author_add'] = 'Добавить другого автора';
+$lang['proj_launch_edition_year'] = 'Год копирайта издания используемого для записи';
+$lang['proj_launch_is_compilation'] = '<span class="red">*</span> Это сборник из нескольких авторов и источников?';
+$lang['proj_launch_translator_add'] = 'Добавить другого переводчика';
+$lang['proj_launch_additional_info_work'] = 'Дополнительные сведения о произведении';
+$lang['proj_launch_additional_info_project'] = 'Дополнительные сведения о проекте';
+$lang['proj_launch_additional_info_soloist'] = 'Сведения о вас (если это ваш первый сольный проект)';
+$lang['proj_launch_choose_lang'] = 'Выберите язык для вашего шаблона';
+$lang['proj_launch_project_solo'] = 'Сольный проект';
+$lang['proj_launch_project_collaborative'] = 'Совместный проект';
+$lang['proj_launch_project_dramatic'] = 'Чтение по ролям';
+$lang['proj_launch_project_poetry_weekly'] = 'Еженедельное стихотворение';
+$lang['proj_launch_project_poetry_fortnightly'] = 'Двухнедельное стихотворение';
+$lang['proj_launch_fill_form'] = 'Заполните эту форму как можно полнее. Скопируйте и вставьте в новый пост форума New Projects Launch Pad.';
+$lang['proj_launch_type_of_project'] = '<span class="red">*</span> Тип проекта';
```

Comment:
I wonder if we could move the HTML out of the translations?

Reply:
I agree that all the `<span class="red">*</span>` HTML should be in the PHP code and not in the translations. I was close to making this change before when doing other work on the Project Launch and language stuff. Maybe now that you have expressed a similar opinion I will go ahead and make this change at some point.